### DISCUSSION TITLE
Open finder folder after download finish

### DISCUSSION THREAD

**Discussion by srujanb** [localsend/localsend#1640] [Discussion (English, International)]
I'm used to airdrop which opens the finder window after the download finishes. It'd be nice to see the same feature with localsend.

**Comment by Nolle10** [comment]
Sounds like a useful feature, have you opened an issue with this idea? @srujanb

**Reply by srujanb** [reply]
I now have. https://github.com/localsend/localsend/issues/1844

**Comment by srujanb** [comment]
Submitted a feature request here. https://github.com/localsend/localsend/issues/1844
